### ISSUE TITLE
Media files missing from Coding anki deck

### ISSUE THREAD

**Issue by gerardo** [donnemartin/interactive-coding-challenges#203]
If you run **Tools -> Check Media** in Anki, the following files are reported as missing:

```
dijkstra.gif
selection_sort.gif
eratosthenes.gif
quicksort_wikipedia.gif
merge_sort.gif
insertion_sort.gif
```

**Comment by donnemartin** [comment]
Hi @gerardo, interesting, I didn't know this check existed.

I think the media should be linked to this repo instead of being embedded in the Anki decks, are these gifs not showing up when you view them in the decks?

**Comment by gerardo** [comment]
Yeah, they're not showing up, just a small blue square with a "?" sign, as Anki shows when the media is not present. 

I checked the HTML code. The images are referenced with a relative path (`<img src="dijkstra.gif" />` for example) instead of being referenced with an absolute URL like you mentioned. That would trigger Anki to download those images and store them on the `media` folder.
